README.md

```diff
@@ -1,7 +1,161 @@
 # Sailfish OS Community repositories: Documentation and issues
 
-OBS projects: 
-- Main project https://build.merproject.org/project/show/sailfishos:chum
-- Testing project for submission of new packages: https://build.merproject.org/project/show/sailfishos:chum:testing
+Sailfish OS Community repositories are providing collection of
+applications, tools, and libraries compiled for different combinations
+of architectures and Sailfish versions.
 
-Use this repository for filing general issues and publishing documentation.
+Ambition is to become the main repository for software distribution on
+Sailfish OS.
+
+When compared to the software distribution via Jolla Store or
+OpenRepos, the software is compiled into RPMs in a reproducible way
+directly from the source. The source used for the compilation is
+available at OBS together with the compiled packages. This is in
+contrast with Jolla Store and OpenRepos where all packages are uploaded
+in binary form without any control over how the binary was compiled.
+
+By collecting the software in a single automated build system, we can
+benefit from collaboration between developers through shared packaging
+of required libraries, reduce duplication of the work by keeping the
+packages up to date, and get a clear overview of missing software.
+
+The repository is located at OBS:
+- https://build.merproject.org/project/show/sailfishos:chum
+
+This repository at GitHub is used for filing general issues and
+publishing documentation.
+
+Origin of "chum": see http://en.wikipedia.org/wiki/Chumming
+
+
+## How to use these repositories: users
+
+At the moment of writing, there is no GUI for selecting packages for
+installation and it has to be done through command line tools
+(`zypper` or `pkcon`).
+
+To use the repositories on your device, users would have to
+- install package `sailfishos-chum`;
```

Comment:
Whats this?

Reply:
it is a small SPEC installing SSU ini file pointing to chum repository. SPEC and INI are included in this PR as well.

Compiled at https://build.merproject.org/package/show/sailfishos:chum/sailfishos-chum and can be distributed at OpenRepos as well. Just to make it easy to add chum repo for users in a way that will be pointing to the correct SFOS version repository

Reply:
Ah, its below!